### ISSUE TITLE
LSynth parts not rendered correctly in native viewer

### ISSUE THREAD

**Issue by laurensvalk** [trevorsandy/lpub3d#447]
### Subject
LSynth parts are not rendered correctly in native viewer, but they look fine with LDView as the renderer.

### Environment
![image](https://user-images.githubusercontent.com/12326241/93722969-150e9000-fb9b-11ea-8885-34c146e603b1.png)

### Configuration
- Downloaded the LPub3D AppImage and ran it on the system above
- Fresh LDraw equal to `complete.zip`, cloned from [here](https://github.com/pybricks/ldraw). It's in `Documents`, so LPub3D automatically finds it on first use.
- LSynth paths are unpacked into the `parts` folder. MLCAD (wine) and LDView (deb package) have no issues seeing them.

### Logs
Not yet made.

### Screenshots

LPub3D with prefered renderer selected as: *Native (default)*
![image](https://user-images.githubusercontent.com/12326241/93723072-ada51000-fb9b-11ea-8642-9b68a821d8d2.png)

LPub3D with preferred render: *LDView*
![image](https://user-images.githubusercontent.com/12326241/93723082-cb727500-fb9b-11ea-84a2-a9fdd795020a.png)

The same applies to the 3D viewer.

### Steps to reproduce

This appears to happen with all LSynth examples I tried. This demo uses [this one](https://github.com/pybricks/ldraw/blob/master/lsynth/examples/ELECTRIC_POWER_FUNCTION_HALF-Constraints.ldr).

### Workaround
Use LDView as preferred render.

**Comment by laurensvalk** [comment]
After struggling with this for a while, I no longer seem to be able to reproduce this. I'll reopen if it occurs again.

**Comment by laurensvalk** [comment]
It seems that 'Archive LSynth Parts` did the trick for the native renderer. Does the native renderer not use the user-specified LDraw path perhaps?

**Comment by trevorsandy** [comment]
Hello Lauren - You are partially correct. If you want to use and LDraw disc library path, you must place your LSynth parts in a folder (other than 'parts' or 'p' under the LDraw unofficial folder and specify this path under search directories in Preferences.  Taking this step will automatically archive the parts when LPub3D starts. To save you all those steps, you can alternatively set the Archive LSynth flag in Preferences.

Cheers,

**Comment by laurensvalk** [comment]
Thanks Trevor. I've used LPub4 a lot in the past ([example](https://www.lego.com/cdn/product-assets/product.bi.additional.extra.pdf/31313_X_RAC3%20TRUCK.pdf)), so it's great to see LPub3D and related tools being developed actively and released for modern platforms.

I've just recently started looking into all these LDraw tools again, this time on Linux. This [repo](https://github.com/pybricks/ldraw) is a small effort to archive those steps so I don't forget where to find everything again in a few years, but it may be useful for others as well.
